templates/macros/https_params.conf

```diff
@@ -0,0 +1,23 @@
+{% macro params() %}
```

Comment:
IMO this should be called `https_params` (like the file) or perhaps even better something entirely different, e.g. `All` (then it would look like this in a template: `{{ https_params.All() }}`) - maybe do it like here for consistency: https://github.com/punktDe/ansible-docker/blob/b485ad00aa39cb9e632f2bb48e2a2b8fbbafabcd/templates/systemd/container.service

Reply:
I have 
```
{% import (role_path + '/../nginx/templates/macros/https_params.conf')|relpath(playbook_dir) as https with context %}
```
in the import and then it gets together as https.params() in the call.
I don't know if I like `All`, this is very generic.
I agree we could name it differently but I didn't come up with a better name yet.